CONTRIBUTING.md

```diff
@@ -1,192 +1,28 @@
-`_configure()` method. This method is responsible for
-
-- Pushing a new Prometheus configuration generated using
-  `_prometheus_config()`.
-- Pushing alert rule files into the Prometheus container using
-  `_set_alerts()`.
-- Pushing the Pebble layer configuration generated using
-  `_prometheus_layer()`. The Prometheus command line in this Pebble
-  layer is generated using `_command()` method.
-- Restarting the workload container or reloading Prometheus
-  configuration and setting charm status.
-
-Generation of the Prometheus configuration is split across multiple
-functions which are invoked by `_prometheus_config()`.
-
-- `_prometheus_global_config()` generates the global
-  configuration. Configuration options are validated using the
-  `_is_valid_timespec()` and `_are_valid_labels()`.
-- `_alerting_config()` generates configuration related to
-  Alertmanager(s), using the Alertmanager charm library.
-- `jobs()` and `alerts()` methods of the `MetricsEndpointConsumer`
-  object.
-
-## Design Choices
-
-This charm manages a [Prometheus](https://prometheus.io) workload and
-in doing so it obtains configuration data through its relations with
-other charms. The key design choice here was to ensure that the
-structure of this data exchanged closely mirrors the format of
-Prometheus' own configuration. This ensures that these relation data
-structures are at least as extensible as Prometheus' own
-configuration. Besides, these data structure would already be familiar
-to Prometheus domain experts.
-
-This Prometheus charm does not support peer relations because as yet
-there is no visible use case for it. In the future should there be a
-use case for
-[federation](https://prometheus.io/docs/prometheus/latest/federation/),
-it may be enabled through the use of peer relations. As a result of
-this decision scaling Prometheus units only results in replication.
-Replicating units will lead to the standard Prometheus "share-nothing"
-replication, in which all units independently scrape all
-targets. However Prometheus scaling is as yet
-[untested](https://github.com/canonical/prometheus-operator/issues/59)
-and hence must be used with caution.
+# Contributing / Hacking
+
+## Intended use case
+
+The charm in this repository is a fork of the 
+[prometheus-k8s-operator](https://github.com/canonical/prometheus-k8s-operator). In addition 
+to the bare Prometheus, this charm includes prometheus-configurer container providing an HTTP API 
+for managing alerting rules. API documentation is available in
+[github](https://github.com/facebookarchive/prometheus-configmanager/blob/main/prometheus/docs/swagger-v1.yml).
+
+## Code contributions
+If you want to propose a new feature, a bug fix or a documentation improvement:
+- Create a new branch from main.
+- Commit and push your changes to this branch.
+- Validate that all Github actions pass.
+- Create a pull request in 
+[github](https://github.com/canonical/charmed-magma-prometheus-operator/pulls).
```

Comment:
Magma?

Reply:
Yeah, that was the original name of the repo.
Fixed.